test/components/css/css_selector_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require "components/css/test_helper"
+Dir["app/components/**/*.rb"].each { |file| require_relative "../../../#{file}" }
+
+class CssSelectorTest < Minitest::Test
+  include Primer::ComponentTestHelpers
+  include Primer::RenderPreview
+
+  IGNORED_SELECTORS = {
+    :global => [/^\d/, ":is", ":root", ":before", ":after", ":hover", ":active", ":disabled", ":focus"],
```

Comment:
I can't think of a case where we would need any selector to have a `:`?

```suggestion
    :global => [/^\d/, ":"],
```